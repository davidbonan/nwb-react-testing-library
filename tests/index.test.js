import React from 'react';
import { render } from '@testing-library/react';
import Component from '../src'

describe("Component", () => {

  test("exists", () => {
    expect(Component).toBeDefined()
  })

  test('matches Snapshot', async () => {
    const { container } = render(<Component />)
    expect(container).toMatchSnapshot()
  })

  test('displays a welcome message', async () => {
    const { getByText } = render(<Component />)
    expect(getByText('Welcome to React components')).toBeInTheDocument()
  })

});