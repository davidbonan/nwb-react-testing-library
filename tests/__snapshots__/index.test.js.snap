// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component matches Snapshot 1`] = `
<div>
  <div>
    <h2>
      Welcome to React components
    </h2>
  </div>
</div>
`;
